corpus/corpus/urls.py

```diff
@@ -21,9 +21,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("accounts/", include("accounts.urls")),
+    path("", include("pages.urls")),
```

Comment:
@anirudhprabhakaran3 if this is before embedathon, then the page never opens the embedathon route, since if goes to the <str:sig_name> in pages.urls and doesn't find a sig named embedathon, hence it should be after all routes such as embedathon right?